Client/Ecosia/UI/EmptyHeader.swift

```diff
@@ -59,7 +59,7 @@ final class EmptyHeader: UITableViewHeaderFooterView, Themeable {
     }
     
     func applyTheme() {
-        image?.image = .init(themed: icon)
+        image?.image = .init(named: icon)?.withRenderingMode(.alwaysTemplate)
```

Comment:
Just as an idea:

You can set `Render as template` directly on the asset catalog inspector.

Reply:
I know, but as icon is a variable I wanna sure that this always the case